.github/workflows/deploy-alpha.yml

```diff
@@ -28,6 +28,9 @@ jobs:
         run: |
           npm ci
           git checkout .
+      - name: Build packages used by others
+        run: |
+          npm run build-essential
```

Comment:
cool,

But, this will need to be added to the master branch in order to work ;)